src/process/style-guideline.rst

```diff
@@ -0,0 +1,486 @@
+entire source text **MUST** be classified as ``crate``.
+
+``system``
+^^^^^^^^^^
+
+A guideline which cannot be checked at the module or crate level and requires the entire
+source text **MUST** be classified as ``system``.
+
+
+``tags``
+--------
+
+The ``tags`` are largely descriptive, not proscriptive means of finding commonality between
+similar guidelines.
+
+Each guideline **MUST** have at least one item listed in ``tags``.
+
+Guideline Content
+-----------------
+
+Each ``guideline`` **MUST** have content which follows the options to give an overview of
+what it covers.
+
+Content **SHOULD** aim to be as short and self-contained as possible, while still explaining
+the scope of the guideline.
+
+Content **SHOULD NOT** cover the rationale for the guideline, which is done in the ``rationale`` section.
+
+Amplification
+^^^^^^^^^^^^^
+
+Guideline Content **MAY** contain a section titled *Amplification* followed by text that provides a more
+precise description of the guideline title. An amplification is normative; if it conflicts with the
+``guideline`` Title, the amplification **MUST** take precedence. This mechanism is convenient as it allows
+a complicated concept to be conveyed using a short Title.
+
+Exception
+^^^^^^^^^
+
+Guideline Content **MAY** contain a section titled *Exception* followed by text that that describes
+situations in which the guideline does not apply. The use of exceptions permits the description of
+some guidelines to be simplified. It is important to note that an exception is a situation in which
+a guideline does not apply. Code that complies with a guideline by virtue of an exception does not
+require a deviation.
+
+``rationale``
+=============
+
+::
+
+      .. rationale::
+         :id: rat_kYiIiW8R2qD1
+         :status: draft
+
+         In debug builds, Rust performs runtime checks for integer overflow and will panic if detected.
+         However, in release builds (with optimizations enabled), integer operations silently wrap
+         around on overflow, creating potential for silent failures and security vulnerabilities.
+         
+         Safety-critical software requires consistent and predictable behavior across all build
+         configurations. Explicit handling of potential overflow conditions improves code clarity,
+         maintainability, and reduces the risk of numerical errors in production.
+
+``rationale`` ``id``
+--------------------
+
+A unique identifier for each rationale. Rationale identifiers **MUST** begin with ``rat_``.
+
+These identifiers are considered **stable** across releases and **MUST NOT** be removed.
+See ``status`` below for more.
+
+**MUST** be generated using the ``generate-guideline-templates.py`` script to ensure
+compliance.
+
+``rationale`` ``status``
+------------------------
+
+The ``status`` option of a ``rationale`` **MUST** match the ``status`` of its parent ``guideline``.
+
+Rationale Content
+-----------------
+
+TODO(pete.levasseur)
```

Comment:
Here's suggestion for kicking off this section.

```suggestion
The rationale **SHOULD** aim to explain to a reader why the guideline is useful in the context of safety critical Rust. The target audience is someone who familiar with Rust, and assumed to have read the contents of the _ The Rust Programming Language_ book, but is otherwise unaware of, or needs to be reminded about specific problematic behaviors that the guideline is attempting to cover. The rationale **MAY** include citations to specific, trusted resources that also explain the guideline, but **MUST** be able to be read coherently without accessing those resources.
```